pymc3/stats.py

```diff
@@ -248,6 +247,9 @@ def loo(trace, model=None, pointwise=False, progressbar=False):
     pointwise: bool
         if True the pointwise predictive accuracy will be returned.
         Default False
+    reff : float
```

Comment:
Could this be computed within the function if not provided?

Reply:
Sure, we can compute it by default.